site/production-checklist.xml

```diff
@@ -100,38 +100,38 @@ limitations under the License.
       <doc:subsection name="resource-limits-ram">
         <doc:heading>Memory</doc:heading>
         <p>
-          By default, RabbitMQ will use up to 40% of available RAM. For nodes
-          that are dedicated to run RabbitMQ, it is often reasonable to raise
-          the limit. However, care should be taken as the OS and file system caches
-          need RAM to operate as well. Failing to do so will have a severe throughput drop due
-          to OS swapping, or even result in the RabbitMQ process terminated by the OS.
+          By default, RabbitMQ will reject any new messages when it detects
```

Comment:
I am not totally sure about `will reject` 
I'd say `won't accept` ( or similar) since reject could be confused with ` basic.reject`